start.py

```diff
@@ -55,7 +56,17 @@ def main(simulation_parameters):
             allagents.do('filobl')
             insurancecustomers.do('check_risk')
 
-        simulation.graphs()
+        if not isTravis:
+            simulation.graphs()
```

Comment:
!!! Sorry, I was late to see the discussion. The test suites in ABCE are currently unittests[1] and have yet to cover for the GUI area of the code. The boolean in `argparse` should have `False` as a default as can be seen in:
```
parser.add_argument('--travis', dest='travis', action='store_true',
                        default=False,
                        help='Flag to disable gui when run in a Travis environment')
```
@x0range  I wired it at `main(simulation_parameters, args.travis)`; I tested on my machine (`uname -o`: `GNU/Linux`, `Python 3.6.1`, this PR) and it launched the GUI. Unless there is an anomaly in the system you are running, it should have run. Yes, the `main` function has `isTravis` as a required argument.

For Protobuf, the default value for bools is false[2]. And yes, in general as well -- a switch is off/0 by default for safety purpose in EE, isn't it?

[1] https://github.com/AB-CE/abce/blob/c93db46ab313f54d6d626725952beac34504fa67/.travis.yml#L47
[2] https://developers.google.com/protocol-buffers/docs/proto